src/metrics.js

```diff
@@ -0,0 +1,81 @@
+const octokit = require('@octokit/rest')() 
+const fs = require('fs');
+ 
+var gitToken = "thisToken"
+ 
+octokit.repos.getForOrg({
+  org: 'octokit',
+  type: 'public'
+})
+ 
+var targetProjectRoot = "C:/Users/SHAWN/data/"
+var github = { "owner": "deepanjali19", "repo": "GitHub-Dashboard", "branch": "master" }
+ 
+downloadGitHubRepo(github, targetProjectRoot)
+ 
+async function downloadGitHubRepo(github, targetDirectory) {
+    console.log(`Installing GitHub Repo ${github.owner}\\${github.repo}`)
+    var repo = github.repo;
+    var path = ''
+    var owner = github.owner
+    var ref = github.commit ? github.commit : (github.tag ? github.tag : (github.branch ? github.branch : 'master'))
+    processGithubDirectory(owner, repo, ref, path, path, targetDirectory)
+}
+
+const binaryExtensions = ['png', 'jpg', 'tiff', 'wav', 'mp3', 'doc', 'pdf']
+var maxSize = 1000000;
+function processGithubDirectory(owner, repo, ref, path, sourceRoot, targetRoot) {
```

Comment:
Above you are using `async/await` style, and here using `Promise.then()` chaining.  I'd suggest picking one or the other.  `async/await` is probably best, so this code could be changed to:

```js
try {
  const result = await octokit.repos.getContent({ "owner": owner, "repo": repo, "path": path, "ref": ref });
  // use result here...
catch (err) {
 // handle errors
}
```